### DISCUSSION TITLE
Changes made to cards in Anki don't save/remain

### DISCUSSION THREAD

**Discussion by krentist** [ObsidianToAnki/Obsidian_to_Anki#424] [Q&A]
For example, making changes like moving cards to different decks or editing cards directly (in Anki) that were originally made in Obsidian then put into Anki through the plugin don't stay. I can't isolate whether this is a problem that arises only from the use of the plugin, or an error in Anki itself, as I make all my cards through Obsidian.

Please help!

**Comment by niposch** [comment]
It has been a while since I've used the plugin but as far as I understand, it's supposed to be a one way sync. Obsidian to Anki.
If I wanted to make a change to a card, it'd need to be made in obsidian and then I'd sync it over to Anki.

**Comment by whatevkev** [comment]
Hi, did you ever figure this out? Fix this?
